### PR TITLE
Credit Card Fraud Detection using Neural Networks

### LINKED ISSUE #275: 💡[FEATURE]: Credit Card Fraud Detection using Neural Networks
**Is your feature request related to a problem? Please describe.**
Frauds of credit cards nowadays are going unnoticed. That is why a credit card fraud detection system is built

**Describe the solution you'd like**
I will build a credit card fraud detection system that can detect credit card frauds and it will be trained using neural networks. I will also add data visualization to the model for better understanding.

### PR DESCRIPTION
**Pull request by Srijansarkar17**
## 🛠️ Related Issue

- Closes #275 

## 📝 Description
This project includes a credit card fraud detection system that can predict whether it is a fraud or not based on various features. The accuracy of the model came out to be 99.94% and the loss was very less. This high accuracy means that the model is very well trained and can predict very efficiently. This project also contains data visualization. 

## 🔍 Type of PR

- [ ] 🐛 Bug fix
- [X] ✨ Feature enhancement
- [X] 📚 Documentation update
- [ ] 🛑 Other (specify): _______________

## 📸 Screenshots / 🎥 Videos (if applicable)

[Attach screenshots or videos]
Screenshot 2024-06-10 113642
Screenshot 2024-06-10 113614
Screenshot 2024-06-10 113552
Screenshot 2024-06-10 113524

## ✅ Checklist

- [X] I have performed a self-review of my code.
- [X] I have read and followed the Contribution Guidelines.
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

## ℹ️ Additional Context
The dataset that was in the form of csv was too big and could not be uploaded. Tried reducing the size but still the file was too big, so i added a link to the dataset in the Readme.md section.

### CODE REVIEW

**Review by Akshat111111 — COMMENTED** [review]
yeah, it works
